.github/workflows/ci.yml

```
@@ -1,73 +1,382 @@
+        npm test -- --watch=false --code-coverage --browsers=ChromeHeadless
+      env:
+        CI: true
 
-      - name: Angular test
-        run: npm run test -- --watch=false --browsers=ChromeHeadless
+    - name: Check coverage threshold and extract metrics
+      run: |
+        # Check if coverage was generated
+        if [ -f "coverage/lcov.info" ]; then
+          echo "✅ Coverage report found"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)